scripts/common/common.sh

```diff
@@ -1036,3 +1036,47 @@ function canonical_image_name() {
     fi
     echo "$image"
 }
+
+# wait_for_running_pods waits for pod(s) in a given namspace to transition to Running status
+function wait_for_running_pods() {
+    local namespace=$1
+    local is_job_controller=0
+    local ns_pods=
+    local status=
+    local containers=
+
+    ns_pods=$(kubectl get pods -n "$namespace" -o jsonpath='{.items[*].metadata.name}')
+
+    if [ -z "$ns_pods" ]; then
+        log "No pods found in namespace $namespace."
+        return 0
+    fi
+
+    for pod in $ns_pods; do
```

Comment:
Would be possible we have a away to pass a timeout so that we could check that for a period of time ?